lib/helpers.ts

```diff
@@ -94,3 +95,22 @@ export function findCurlyBracedDirectives(directive: string, str: string) {
 
   return matches;
 }
+
+export interface LintRollerTsCheckConfig {
+  defaultImports?: string[];
+  typings?: string[];
+}
+
+export interface LintRollerConfig {
+  'markdown-ts-check'?: LintRollerTsCheckConfig;
+}
+
+export function loadConfig(path: string) {
+  if (!fs.existsSync(path)) {
+    return undefined;
+  }
+
+  const config = fs.readFileSync(path, 'utf8');
+
+  return JSON.parse(config) as LintRollerConfig;
```

Comment:
Any value in a try/catch here? Then again, an error is an error and it's probably usable enough as it is

Reply:
Made it more user friendly.